### PR TITLE
Attr support for proof restrictions

### PR DESCRIPTION
**Pull request by ianco**
Signed-off-by: Ian Costanzo <ian@anon-solutions.ca>

### CODE REVIEW

**Comment by sklump** [comment]
Hold off on this until Monday please - I'd like to take a deeper look

**Comment by ianco** [comment]
@sklump I took a crack at updating the schema so we could keep the schema validations on the restrictions however I couldn't get it to work.  You can see the updated schema code here:

https://github.com/ianco/aries-cloudagent-python/compare/attr_value_restrictions...ianco:attr_value_restrictions_dev?expand=1

File: aries_cloudagent/protocols/present_proof/v1_0/routes.py [hunk 1]
```diff
@@ -206,7 +212,7 @@ class IndyProofReqPredSpecSchema(Schema):
     )
     p_value = fields.Integer(description="Threshold value", required=True)
     restrictions = fields.List(
-        fields.Nested(IndyProofReqSpecRestrictionsSchema()),
+        fields.Dict(),  # fields.Nested(IndyProofReqSpecRestrictionsSchema()),
```
Comment (sklump) [inline aries_cloudagent/protocols/present_proof/v1_0/routes.py:215]: Let me have a crack at it, I am just off vacation and full of hubris

**Comment by sklump** [comment]
I'm tentatively closing this one, please follow on #587. No editorial reason but a safeguard to make sure both don't get merged. If #587 is the wrong idea, we can open this one again instead.

**Pull request closed without merging**